code/modules/jobs/job_types/botanist.dm

```diff
@@ -34,4 +34,16 @@
 	satchel = /obj/item/storage/backpack/satchel/hyd
 	courierbag = /obj/item/storage/backpack/messenger/hyd
 
+//shiptest!!!!!!!!!!
+/datum/outfit/job/botanist/syndicate/nsv
+	name = "Botanist-Chemist (NSV-M)"
+
+	uniform = /obj/item/clothing/under/syndicate/skirt
```

Comment:
jumpsuit prefs exist for a reason, I know it's funny but you probably expected this

Reply:
dont see the issue

Reply:
not being merged until you fix it and I know you know what I mean

Reply:
and that's a promise

Reply:
fair enough

Reply:
@MarkSuckerberg fixed it, they are just regular toitlenecks now